packages/freezed/lib/src/templates/copy_with.dart

```diff
@@ -61,17 +61,19 @@ ${_abstractDeepCopyMethods().join()}
   String get abstractCopyWithGetter {
     if (cloneableProperties.isEmpty) return '';
 
-    return _maybeOverride(
-      '''
-@JsonKey(ignore: true)
+    return '''
```

Comment:
Sounds like _maybeOverride should have an optional `doc` parameter. Otherwise LGTM